package.json

```diff
@@ -69,14 +69,20 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "node ./test/runTest.js"
+    "test": "node ./test/runTest.js",
```

Comment:
Can you explain how it works?

Reply:
I'm not sure what you mean. The scripts can be run by `npm run`. Then esbuild do the compressing. I didn't config anything.